src/compilerServices/clangService.ts

```diff
@@ -31,23 +31,26 @@ export class ClangService implements CompilerService {
 
   async compile(input: ServiceInput): Promise<ServiceOutput> {
     const files = Object.values(input.files);
-    if (files.length !== 1) {
-      throw new Error(`Supporting compilation of a single file, but ${files.length} file(s) found`);
-    }
     const [ fileRef ] = Object.keys(input.files);
-    const src = files[0].content;
     const from = this.lang;
+    const inputFile=Object.keys(input.files);
```

Comment:
very poor code formatting and identifiers names -- hard to read the change

Reply:
I should be careful to identifiers names and formatting the codes.